js/modules/inputValidation.js

```diff
@@ -0,0 +1,108 @@
+const forms = Array.from(document.forms);
+let isInputHandlerAdded = false;
+
+const REGEXP = {
+	NAME: /^[а-яА-ЯёЁa-zA-ZЁёЇїІіЄєҐґ']+$/,
```

Comment:
Looks like the name can be with infinite length, is it okay?

Reply:
Done.